pkg/systemd/services/services.jsx

```diff
@@ -343,6 +331,15 @@ class ServicesPageBody extends React.Component {
         return true;
     }
 
+    loadPinnedUnits() {
+        try {
+            this.setState({ pinnedUnits: JSON.parse(localStorage.getItem('systemd:pinnedUnits')) || [] });
+        } catch (err) {
+            console.warn("exception while parsing systemd:pinnedUnits", err);
+            this.setState({ pinnedUnits: [] });
+        }
```

Comment:
These 4 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18416-20230227-182216-6d0ed185-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)